pyo3-ffi/src/lib.rs

```diff
@@ -364,6 +364,7 @@ mod ceval;
 #[cfg(Py_LIMITED_API)]
 mod code;
 mod codecs;
+pub mod compat;
```

Comment:
I think because this isn't a CPython header and instead a real module, it might be worth moving this to a separate location in the file, maybe around line 292 just under the `use` statement.